server/models/eventSchema.js

```diff
@@ -4,13 +4,14 @@ const Schema = mongoose.Schema;
 
 const eventSchema = new Schema(
     {
-      	creator: {type: Schema.Types.ObjectId, ref: 'User', required: true},
-        name: {type: String, required: true},
-        date: {type: Date, rrequired: true},
+        creator: {type: Schema.Types.ObjectId, ref: 'User', required: true},
+        title: {type: String, required: true},
+        date: {type: Date, required: true},
         link: {type: String, required: true},
         description: {type: String, required: true},
         school: {type: Schema.Types.ObjectId, ref: 'School', required: false},
-        years: [{type: Number, required: false}]
+        startYear: {type: Number, required: false},
+        endYear: {type: Number, required: false}
```

Comment:
What do we store in the database if we want everyone from the school to be invited?

Reply:
I think the default is use all years unless specifies otherwise because neither `startYear` nor `endYear` are required. This also allows for a bit of flexibility with the year ranges so when creating events, you could also specify you want to include student after x year or student before y year.

Reply:
Cool, sounds good!